models/json-file-schema.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var jsonObjectSchema = new Schema({
+  CategoryID: {
```

Comment:
Please follow the naming conventions. for eg: **CategoryID** should be **categoryID**